run_iwyu_tests.py

```diff
@@ -124,7 +125,16 @@ def PrintLoadedTestsAndFiles():
   (runner_args, _) = parser.parse_known_args(unittest_args)
 
   if runner_args.run_test_file:
-    exit(TestIwyuOnRelevantFiles(runner_args.run_test_file))
+    try:
+      r = TestIwyuOnRelevantFiles(runner_args.run_test_file)
+    except unittest.SkipTest as e:
+      # Catch the skip test exception that unittest would normally
+      # handle. Instead set a return code that informs Ctest that the
```

Comment:
Maybe mention that we've aligned the return code with GNU?

Reply:
Done